apps/files/src/components/FileList.vue

```diff
@@ -301,6 +308,32 @@ export default {
       }
 
       return `files-file-list-action-button-${id}`
+    },
+
+    $_resizeHeader () {
+      this.$nextTick(() => {
+        const headerRow = this.$refs.headerRow
+        const firstRow = this.$refs.firstRow
+        if (headerRow && firstRow) {
+          const headerColumns = Array.from(headerRow.$el.childNodes).filter(n => n.nodeType === Node.ELEMENT_NODE)
+          const rowColumns = Array.from(firstRow.$el.childNodes).filter(n => n.nodeType === Node.ELEMENT_NODE)
```

Comment:
You could rename `rowColumns` to `bodyColumns`, just to stay in the wording context of tables.

Reply:
Also in all variables and comments below.